### ISSUE TITLE
Import Settings File/ Multiple Profiles

### ISSUE THREAD

**Issue by catomic** [runelite/runelite#11361]
So I recently made a pure and I would like to have different settings than the main account. I realize I could just use two different email addresses to log into runelite with but I found you can redirect the runelite shortcut application to use a different settings.properties file. So in my trials to get this to work I discovered regardless of what I changed it would auto-update for both shortcuts. Here are the filepaths used.
Runelite shortcut used for main: 
C:\Users\xxx\AppData\Local\RuneLite\RuneLite.exe --c --config="settings.properties"

Runelite shortcut used for pure:
C:\Users\xxx\AppData\Local\RuneLite\RuneLite.exe --c --config="pure.properties"

After some tinkering I realized the problem was that I was logged in on runelite, therefor whenever I changed settings it would save for both accounts rendering the different properties files useless. Easy solution would be to just log out, well not so much. I had a new problem now when I log out everything worked as expected the settings were different. But when I tried to log in on only one runelite account, it would log in for both shortcuts. 

This is where I am stuck. The original easy solution of having two seperate runelite email addresses registered would be adding an extra step, every time i open a different account (which is often) i would have to log out and log back in to another email. This is tedious. I am trying to figure out if there is a way in the settings.properties file or some other way to have runelite logged in on only one shortcut. This would make it possible for me to have all the settings exact while logged out of runelite on the pure, and therefor any changes i make on the main i would have to re-make on the pure as well which is fine. This way my main account is cloud synced in the event of a drive failure or i use a different computer or whatever the case is, but I dont have to worry about settings conflicting with the pure. 

Does anyone have any ideas, or info that I might be missing that could make this much easier. I realize it may be somewhat confusing what I am trying to accomplish. So if I need to clarify something please let me know.

TLDR: I am trying to have two shortcuts for each account that have independently different settings.

Side Note:
When I log out of my cloud synched account, all of the settings i have configured are removed such as GPU, tile markers etc... I was under the impression that the cloud only synchs the settings with the settings.properties file, so if you were to log in on a fresh runelite install with no configuration, it would apply all your settings and you could log out and it would save them locally. This isnt the case, so how would I go about importing these cloud settings locally to say the pure.properties shortcut, obviously I could manually go through each setting and check them but that would be annoying to do. I looked on the runelite website while logged in and while you can download your loottracker, ge history and bosslogs i could not find a settings download.

**Comment by loldudester** [comment]
Use the --session argument, rather than --config. Point each shortcut to a different session file (it will generate one the first time you run if the one you point it to doesn't exist), and log into different Runelite accounts on each shortcut (or simply don't log in on one of them). When you run each shortcut after that they will launch with the correct Runelite account each time.

**Comment by loldudester** [comment]
As for copying your main account's settings for your pure, they are saved in the .runelite\profiles\ directory when they sync from the server.

**Comment by catomic** [comment]
So I found the sync settings however copying and renaming it to pure.properties is not saving in the settings, any ideas.

Here is the new path example, also I only see one session file, so how would I go about making a second one if its not doing it automatically. 

C:\Users\xxx\AppData\Local\RuneLite\RuneLite.exe --c --session="pure.properties"

**Comment by ypperlig** [comment]
As this is not a support forum you should join #support on the discord instead for help.

Regardless, the following wiki article shows how to do this:
[https://github.com/runelite/runelite/wiki/General-Features#playing-with-multiple-accounts](https://github.com/runelite/runelite/wiki/General-Features#playing-with-multiple-accounts)
